python/dask_cudf/dask_cudf/io/orc.py

```diff
@@ -81,3 +81,75 @@ def read_orc(path, columns=None, storage_options=None, **kwargs):
 
     divisions = [None] * (len(dsk) + 1)
     return dd.core.new_dd_object(dsk, name, meta, divisions)
+
+
+def _write_partition(df, path, fs, filename, compression=None):
+    return cudf.io.to_orc(
+        df, fs.sep.join([path, filename]), compression=compression
+    )
```

Comment:
By convention many Dask writers return the filenames of each partition.  Perhaps we should do that here?

Alternatively, we might drop this function entirely, and just pass around `cudf.io.to_orc` rather than have the additional wrapper.

Reply:
I'm good with either approach - The only reason I used a separate function was to preserve a clear mapping to the general dask solution (to be implementated after)

Reply:
Also, if we keep this function, we should probably name it something like `write_orc`.  This is the name that will show up in the dashboard.  Write partition is nice, but something with orc in the name would be nice too.

Reply:
Let's keep this function and have it return the final path

Reply:
Sounds good - I changed the name to `write_orc_partition`